src/ProfilesList.json

```
@@ -438,6 +438,7 @@
   "tainguyenphat74.json",
   "Saipradyumna Goud Chiragoni.json",
   "dylan-dot-c.json",
+  "Omkar-ctrl.json",
```

```suggestion
  "omkar-ctrl.json",
```

@Omkar-ctrl  Make sure the file name matches the file name of your profile you created.